systemd/ceph-volume@.service

```diff
@@ -5,10 +5,12 @@ Wants=local-fs.target
 
 [Service]
 Type=oneshot
-KillMode=none
-Environment=CEPH_VOLUME_TIMEOUT=10000
-ExecStart=/bin/sh -c 'timeout $CEPH_VOLUME_TIMEOUT /usr/sbin/ceph-volume-systemd %i'
-TimeoutSec=0
+
+EnvironmentFile=-/etc/default/ceph
+Environment=CLUSTER=ceph
+
+ExecStart=/usr/sbin/ceph-volume-systemd --cluster ${CLUSTER} --osd-id %i activate
+ExecStart=/usr/sbin/ceph-volume-systemd --cluster ${CLUSTER} --osd-id %i deactivate
```

Comment:
This will run serially when the unit is triggered, so activate and immediate deactivate?

Reply:
Sorry, second line is supposed to be a ExecStop=